llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -205,6 +210,53 @@ class Constant : public sandboxir::User {
 #endif
 };
 
+/// The BasicBlock::iterator.
+class BBIterator {
+public:
+  using difference_type = std::ptrdiff_t;
+  using value_type = Instruction;
+  using pointer = value_type *;
+  using reference = value_type &;
+  using iterator_category = std::bidirectional_iterator_tag;
+
+private:
+  llvm::BasicBlock *BB;
+  llvm::BasicBlock::iterator It;
+  Context *Ctx;
+  pointer getI(llvm::BasicBlock::iterator It) const;
```

Comment:
can `getI` be more descriptive? (I assume this is `getIterator`?)

Reply:
Renamed it to `getInstr()`.